blueoil/common.py

```diff
@@ -57,3 +60,48 @@ def get_color_map(length):
     # This function generate arbitrary length color map.
     color_map = COLOR_MAP * int(math.ceil(length / len(COLOR_MAP)))
     return color_map[:length]
+
+
+# For replacing the Matplotlib Jet colormap, we use the Turbo color map
+# https://ai.googleblog.com/2019/08/turbo-improved-rainbow-colormap-for.html
+# The colormap allows for a large number of quantization levels:
+# https://github.com/blue-oil/blueoil/tree/master/docs/_static/turbo_cmap.png
```

Comment:
Missing link. Do you have plan to add image to the documentation?

Reply:
@iizukak Yes, I will add that image and upload to the commit.